dev_guide/integrating_external_services.adoc

```diff
@@ -369,18 +366,19 @@ by setting environment variables in the appropriate containers.
 
 ====
 
-=== External SaaS Provider Environment Variables
+*External SaaS Provider Environment Variables*
 
-Using an external service in your application is just like using an internal service. Your application is
-injected with environment variables for the service and the additional environment variables with the credentials
-described above. In the following example, the 'helloworld' container receives the following environment variables:
+Like when using an internal service, your application is injected with
```

Comment:
Perhaps "Like when...service" would be better expressed as "Similarly to using...service" ?